### DISCUSSION TITLE
Dressing.nvim

### DISCUSSION THREAD

**Discussion by TobinPalmer** [nvim-tree/nvim-tree.lua#2128] [Q&A] [answered]
With nvim-tree, is there a way to, A: Prevent closing the tree, and B: Align the cursor when using `vim.ui.input()` OR to just disable dressing.nvim when all `vim.ui.select()` actions in nvim-tree. This is what I currently have 
![image](https://user-images.githubusercontent.com/99927862/231199000-d2c27631-86a7-4932-aebb-95ee0d731b55.png)
![image](https://user-images.githubusercontent.com/99927862/231199074-e0246fb8-6ec0-4d34-a63e-38764fc61c16.png)
As you can see, dressing.nvim just appears in the upper right corner.

**Comment by alex-courtis** [comment]
I'm not familiar with dressing.nvim - what does it do? What are you expecting from it?

What is the action in nvim-tree you are executing to cause a select?

It looks like you are using a floating nvim-tree window. What is the behaviour when it is not floating?

**Reply by TobinPalmer** [reply]
dressing the nvim is a library that styles vim.ui.input() and other ui elements of nvim.
https://github.com/stevearc/dressing.nvim

**Comment by TobinPalmer** [comment, marked as answer]
fixed:
```
require('dressing').setup {
  input = {
    get_config = function()
      if vim.api.nvim_buf_get_option(0, 'filetype') == 'NvimTree' then
        return { enabled = false }
      end
    end,
  },
}
```
